packages/embed/e2e/spec/popup.spec.ts

```diff
@@ -0,0 +1,9 @@
+describe('Popup', () => {
+  it('should open popup', () => {
+    cy.visit('/iframe.html?id=embed-popup--popup&viewMode=story')
```

Comment:
Storybook actually renders the story in an iframe. We can open the story directly without all the fancy UI around it and without iframe (Cypress can access inside of iframes only with disabled Chrome web security). This way we can reuse Storybook stories for functional testing with no extra effort.